src/Parachain.sol

```diff
@@ -41,9 +41,12 @@ abstract contract Parachain {
             hex"0C", // fixed call index within pallet: 12
             _reporter, // account id of reporter on target parachain
             bytes32(reverse(_amount)), // amount
-            bytes20(_staker) // staker
```

Comment:
Why no longer casting as `bytes20`? is this already happening? We just need to be sure, because that's the type expected on the oracle consumer parachain side.

Reply:
addresses are already 20 bytes by default, you don't have to cast into bytes20 again